user/user.h

```diff
@@ -1,41 +1,46 @@
 struct stat;
 
+#define stdin 0
+#define stdout 1
+#define stderr 2
+
 // system calls
 int fork(void);
 int exit(int) __attribute__((noreturn));
-int wait(int*);
-int pipe(int*);
-int write(int, const void*, int);
-int read(int, void*, int);
+int wait(int *);
+int pipe(int *);
+int write(int, const void *, int);
+int read(int, void *, int);
 int close(int);
 int kill(int);
-int exec(const char*, char**);
-int open(const char*, int);
-int mknod(const char*, short, short);
-int unlink(const char*);
-int fstat(int fd, struct stat*);
-int link(const char*, const char*);
-int mkdir(const char*);
-int chdir(const char*);
+int exec(const char *, char **);
+int open(const char *, int);
+int mknod(const char *, short, short);
+int unlink(const char *);
+int fstat(int fd, struct stat *);
+int link(const char *, const char *);
+int mkdir(const char *);
+int chdir(const char *);
 int dup(int);
 int getpid(void);
-char* sbrk(int);
+char *sbrk(int);
 int sleep(int);
```

Comment:
while i agree unsigned int is better (and its what linux uses)
the xv6 kernel declares that its an int, and sys_sleep expects an int argument.

Reply:
declaring it as an unsigned int would actually cause more problems since then passing sleep(MAX_INT + 1) would not sleep at all, while being a perfectly legal unsigned int value